pkgs/by-name/um/umu-launcher-unwrapped/no-umu-version-json.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/Makefile.in b/Makefile.in
+index b82053d..37db8c9 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -50,7 +50,7 @@ fix_shebangs:
+ umu/umu_version.json: umu/umu_version.json.in
+ 	$(info :: Updating $(@) )
+ 	cp $(<) $(<).tmp
+-	sed 's|##UMU_VERSION##|$(shell git describe --always --long --tags)|g' -i $(<).tmp
++	sed 's|##UMU_VERSION##|@version@|g' -i $(<).tmp
+ 	mv $(<).tmp $(@)
+ 
+ .PHONY: version
+@@ -151,7 +151,7 @@ clean:
+ 	@rm -rf -v $(OBJDIR) umu/umu_version.json ./$(RELEASEDIR) $(RELEASEDIR).tar.gz
+ 
+ 
+-RELEASEDIR := $(PROJECT)-$(shell git describe --abbrev=0)
++RELEASEDIR := $(PROJECT)-@version@
```

Comment:
This patch could be avoided by overriding `RELEASEDIR` explicitly in `makeFlags`.

If it matters, we know `$(PROJECT)` should always be `umu-launcher`. But I think for our purposes `RELEASEDIR` could be any string?